src/User/UserContainer.jsx

```diff
@@ -1,43 +1,54 @@
 import React, { useState, useEffect } from "react";
 import { Switch, Route, useParams } from "react-router-dom";
 import ProfilePage from "./ProfilePage";
-import { getUserById } from "../api.js";
 import UpdateProfile from "./UpdateProfile";
+import { getUserById } from "../api.js";
 
-const UserContainer = ({ token }) => {
-  const [user, setUser] = useState(null);
-  const [isLoading, setIsLoading] = useState(true);
-  const [error, setError] = useState(false);
-  const { id } = useParams();
+const UserContainer = ({ sessionToken }) => {
```

Comment:
I think you guys need to configure your settings in VScode so that they're all the same, lots of random whitespace changes makes it a bit harder to review whats change